requirements.txt

```diff
@@ -0,0 +1,2 @@
+pysam
+biopython
```

Comment:
As far as I understand, Conda can also be used to install both Pysam and Biopython. So if we do want to continue using Conda, maybe we want to do it consistently and install _everything_ using it. Alternatively, we could scrap Conda entirely and install everything using standalone commands/PIP. @tcezard @BaronKoy what are your thoughts?

Reply:
I'm actually starting to think that Conda isn't such a good tools after all. We've had a number of issues with conflicting tool dependency. It is forcing us to use specific tool version (such as `samtools 1.9` instead of `samtools 1.10`). I still think it has some value for simplifying the setup of the test but I would not advocate we use it on our deployment.
I would defer this to a different ticket though.

Reply:
In case we're not going to use Conda in production, I think we shouldn't use it for tests as well. But agree this should be done in a separate ticket. Did you already create one? Or I can go ahead and create one as well, just don't want to end up with duplicates.

Since migrating away from Conda is not a huge task, maybe what we could do is have a one (or a small number) of aggregate follow-up ticket(s) with several small changes.

Reply:
Will follow up in https://www.ebi.ac.uk/panda/jira/browse/EVA-2001.